### PR TITLE
feat: add fraxtal config

### PR DESCRIPTION
**Pull request by blockchainguyy**
JIRA: https://axelarnetwork.atlassian.net/browse/AXE-3075

### CODE REVIEW

**Review by re1ro — APPROVED** [review]

**Review by deanamiel — APPROVED** [review]

File: axelar-chains-config/info/mainnet.json [hunk 1]
```diff
@@ -2007,6 +2007,43 @@
...
+    "fraxtal": {
+      "name": "Fraxtal",
+      "id": "fraxtal",
+      "axelarId": "fraxtal",
+      "chainId": 252,
+      "rpc": "https://rpc.frax.com",
+      "tokenSymbol": "frxETH",
+      "confirmations": 2,
+      "gasOptions": {
+        "gasLimit": 8000000
```
Comment (canhtrinh) [inline axelar-chains-config/info/mainnet.json:2020]: this was set to 3mm originally, but i pushed to 8mm as set on other chains. not sure if 3mm was intentional

File: axelar-chains-config/info/mainnet.json [hunk 2]
```diff
@@ -2007,6 +2007,43 @@
...
       }
+    },
+    "fraxtal": {
+      "name": "Fraxtal",
+      "id": "fraxtal",
+      "axelarId": "fraxtal",
+      "chainId": 252,
+      "rpc": "https://rpc.frax.com",
+      "tokenSymbol": "frxETH",
+      "confirmations": 2,
```
Comment (canhtrinh) [inline axelar-chains-config/info/mainnet.json:2018]: i changed this from 1 to 2 as with other L2 chains. not sure if  1 was intentional

Comment (milapsheth) [reply axelar-chains-config/info/mainnet.json:2018]: 2 is good just to confirm there's no immediate reorg

File: axelar-chains-config/info/mainnet.json (hunk 1 above)
Comment (blockchainguyy) [reply axelar-chains-config/info/mainnet.json:2020]: 3m was able to run all tests successfully for me and 8m resulted in the lack of funds for execution, therefore committed 8m. Good with 8m here for consistency.

**Pull request merged**